client/src/main/scala/caliban/client/ArgEncoder.scala

```diff
@@ -42,6 +51,8 @@ object ArgEncoder {
 
   implicit val unit: ArgEncoder[Unit] = (_: Unit) => __ObjectValue(Nil)
 
+  implicit val uuid: ArgEncoder[UUID] = (value: UUID) => __UUIDValue(value)
```

Comment:
You could just use `__StringValue` here, no need to add a new primitive `__UUIDValue`.

Reply:
Done